ChangeLog.md

```diff
@@ -3,6 +3,317 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-09
+
+
+### Key Changes
+
+- Reworked supply mechanics (details below)
+- Reworked weapon part refinements
+- Reworked ship repair: Drydocks, Damage Control techs,
+Robotic hulls self repair (details below)
+- Substantial GUI improvements
+ - Substantial improvements to GUI responsiveness and FPS
+ - Made GUI more configurable
+- Many small additions/changes to GUI rendering, layout, functionality,
+customization options, and displayed information that collectively
+substantially improve usability, responsiveness, and system resource usage.
```

Comment:
@geoffthemedio, to save time (I'm deferring the RC1 builds here), just commit the corrections/adjustments as you see fit.